src/index.ts

```diff
@@ -56,7 +56,7 @@ async function tailwindCSSModule (moduleOptions) {
     logger.info(`Merging Tailwind config from ~/${relative(this.options.srcDir, configPath)}`)
     // Transform purge option from Array to object with { content }
     if (Array.isArray(tailwindConfig.purge)) {
-      tailwindConfig.purge = { content: tailwindConfig.purge }
+      tailwindConfig.content = tailwindConfig.purge
```

Comment:
Please check this part especially for mistakes on my end! The tests run fine, tough.

Reply:
This looks good on my end! :confetti_ball:

Reply:
I think this whole clause of _transforming the purge option from Array to object_ is no longer required, the users should change this in their tailwind config according to the tailwind guide.